src/lsst/cm/tools/db/sqlalch_interface.py

```diff
@@ -489,15 +490,29 @@ def daemon(
             self.launch_jobs(LevelEnum.campaign, db_id, max_running)
             self.check(LevelEnum.campaign, db_id)
             if verbose:
-                self.print_table(sys.stdout, TableEnum.step)
-                self.print_table(sys.stdout, TableEnum.group)
-                self.print_table(sys.stdout, TableEnum.workflow)
+                if log_file is None:
```

Comment:
How about:

```Python
with open(log_file, "a") if log_file else nullcontext(sys.stdout) as log_stream:
    self.print_table(log_stream, TableEnum.step)
    self.print_table(log_stream, TableEnum.group)
    self.print_table(log_stream, TableEnum.workflow)
```

...here, instead, to avoid repeated code blocks that can go out of sync?

Reply:
🤸‍♂️ great idea! 🎆